service/controller/chart/v1/key/key.go

```diff
@@ -0,0 +1,33 @@
+package key
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	versionBundleAnnotation = "giantswarm.io/version-bundle"
```

Comment:
Since the chart CRs are customer facing we use this annotation instead of having it under Spec.

See https://github.com/giantswarm/fmt/pull/36.